app/views/moderations/actions_panel.html.erb

```diff
@@ -175,8 +182,9 @@
       </a>
     </div>
 
-    <% if current_user.any_admin? && @moderatable.published %>
-      <button aria-haspopup="true" aria-expanded="false" aria-controls="admin-action-options" class="other-things-btn admin-action" type="button" data-other-things-type="admin-action" aria-label="<%= t("views.moderations.actions.admin.aria_label") %>">
+    <%# Hiding this Admin Actions panel until Flag User, Unpublish All Posts and Suspend User are moved into it  %>
```

Comment:
Right now, there is nothing in the Admin Actions dropdown, so I am hiding it. That will change when I move the actions Flag User, Suspend User and Unpublish All Posts into this dropdown.